src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -316,9 +317,14 @@ private void recoverCommitlogAndCompleteSetup() {
         {
             StorageService.instance.initServer();
         }
+        catch (BootstrappingSafetyException e)
+        {
+            logger.error(e.getMessage() + "\nNon-fatal bootstrap error. Server will continue but is disabled and without metrics.");
```

Comment:
nit: These could probably be replaced with:
```
logger.error("Non-fatal bootstrap error. Server will continue but is disabled and without metrics.", e);
```

Reply:
Good catch, refactored.